TestCases/hybrid_regression.py

```diff
@@ -436,7 +436,7 @@ def main():
     square_cylinder.cfg_dir   = "unsteady/square_cylinder"
     square_cylinder.cfg_file  = "turb_square.cfg"
     square_cylinder.test_iter = 3
-    square_cylinder.test_vals = [-1.162564, 0.066401, 1.399788, 2.220402]
+    square_cylinder.test_vals = [-1.162564, 0.066401, 1.399788, 2.220402, 1.399743, 2.218605]
```

Comment:
I'll prob have to adapt this according to what the workflow says, as the available values are already slightly different to the MPI-only values and I am too lazy to set up a compilation with OMP

Reply:
```suggestion
    square_cylinder.test_vals = [-1.162564, 0.066401, 1.399788, 2.220402, 1.399743, 2.218603]
```
Tiny change to have exactly the workflow values. The Test didn't fail as it was the difference was below the threshold.